### ISSUE TITLE
Bidimensional SFS

### ISSUE THREAD

**Issue by adria-boada** [mfumagalli/ngsJulia#9]
Good evening,

`ngsPool` provides `poolSFS.R`, a script which estimates a one dimensional SFS from the `SAF.GZ` output file. Is it possible for other software to read pairs of "sample allele frequency log-likelihoods" from `ngsJulia` and estimate a bidimensional SFS?

I do know about an implementation in ANGSD: ["Fast and accurate estimation of multidimensional site frequency spectra from low-coverage high-throughput sequencing data"](https://doi.org/10.1093/gigascience/giac032). Would it be possible to translate the SAF files from ngsJulia so that they can be read by ANGSD? I have been unable to accomplish so.

[ANGSD documentation](https://www.popgen.dk/angsd/index.php/RealSFS)

Requirements, at the end of the article cited above:
> The program is implemented in a fast multi-threaded C++ program and takes as input either BAM/CRAM files or BCF/VCF files containing genotype likelihood files as produced from standard tools such as GATK or SAMtools.

Kind regards,
Adrià.

**Comment by mfumagalli** [comment]
Good point. Can you please contact me separately via email to continue this conversation (and about the other points you raised) separately?
________________________________
From: adria-boada ***@***.***>
Sent: 10 November 2024 7:33 PM
To: mfumagalli/ngsJulia ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [mfumagalli/ngsJulia] Bidimensional SFS (Issue #9)


Good evening,

ngsPool provides poolSFS.R, a script which estimates a one dimensional SFS from the SAF.GZ output file. Could you please recommend other software to read pairs of "sample allele frequency log-likelihoods" and estimate a bidimensional SFS?

I do know about an implementation in ANGSD: "Fast and accurate estimation of multidimensional site frequency spectra from low-coverage high-throughput sequencing data"<https://doi.org/10.1093/gigascience/giac032>. Would it be possible to translate the SAF files from ngsJulia so that they can be read by ANGSD? I have been unable to accomplish so.

Kind regards,
Adrià.
